FlightReservation/components/mainScreen.js

```diff
@@ -0,0 +1,96 @@
+import React, {useState} from 'react';
+import {View, Text} from 'react-native';
+
```

Comment:
```suggestion
```

Reply:
done

Reply:
We already corrected all the suggestions you have done to us. Except for the one about "proptypes", as we already continue with another issue and a pull request, and that one has more changes, and obviously the same issues, we are going to add the "proptypes" in that issue, so we would try to avoid merge conflicts.